apps/core-app/src/main/modules/box-tool/addon/preview/preview-registry.ts

```
@@ -1,65 +1,44 @@
-      } catch (error) {
-        if ((error as DOMException).name === 'AbortError') {
-          return null
-        }
-        previewRegistryLog.error('Ability failed', {
-          meta: {
-            abilityId: ability.id
-          },
-          error
-        })
+export function registerPreviewAbility(ability: PreviewAbility): void {
+  const existing = previewAbilityRegistry.list().some((item) => item.id === ability.id)
```

ESLint

[error] Unexpected parentheses around single function argument having a body with no curly braces.

(style/arrow-parens)